iron-ajax.html

```diff
@@ -353,13 +353,15 @@
         contentType = 'application/x-www-form-urlencoded';
       }
       if (contentType) {
-        headers['Content-Type'] = contentType;
+        headers['content-type'] = contentType;
       }
       var header;
 
       if (this.headers instanceof Object) {
         for (header in this.headers) {
-          headers[header] = this.headers[header].toString();
+          // Normalize headers in lower case to make it easier for iron-request
+          // to reason about them.
+          headers[header.toLowerCase()] = this.headers[header].toString();
```

Comment:
@rictic couldn't the lower casing have been done in iron-request since we already do a check there? And log a warn message instead of an error? 
I'm relying on iron-request and for me this was a breaking change introduced by a simple patch version which kind of defeats the purpose of semver.